hi.js

```diff
@@ -82,7 +82,7 @@ settings.import इम्पोर्ट
 settings.export ऐक्सपोर्ट
 settings.none कोई सेटिंग नहीं मिली
 settings.backup बैकअप सेटिंग्स
-settings.controls.change Keybinds
+settings.controls.change कीबइंडस 
```

Comment:
i think it would be better if left as it is, as it isnt precise

Reply:
Sure